clouddriver-elasticsearch-aws/src/main/kotlin/com/netflix/spinnaker/clouddriver/elasticsearch/ElasticSearchClient.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.elasticsearch
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.clouddriver.elasticsearch.model.ElasticSearchException
+import com.netflix.spinnaker.clouddriver.elasticsearch.model.ServerGroupModel
+import io.searchbox.client.JestClient
+import io.searchbox.core.Bulk
+import io.searchbox.core.BulkResult
+import io.searchbox.core.Index
+import io.searchbox.indices.CreateIndex
+import io.searchbox.indices.DeleteIndex
+import io.searchbox.indices.aliases.AddAliasMapping
+import io.searchbox.indices.aliases.GetAliases
+import io.searchbox.indices.aliases.ModifyAliases
+import java.io.IOException
+
+class ElasticSearchClient(val objectMapper : ObjectMapper, val jestClient: JestClient) {
```

Comment:
might want to specify `private val` for those as without that they are publicly accessible outside this class.

Reply:
good call